### PR TITLE
Fixed placeholder cursor and cleanup

### PR DESCRIPTION
**Pull request by jorilallo**

### CODE REVIEW

File: src/components/Placeholder.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 import styled from "styled-components";
 
-export default styled.span`
-  position: absolute;
-  top: 0;
-  visibility: hidden;
+export default styled.span.attrs({
+  contentEditable: false,
+})`
+  display: inline-block;
```
Comment (jorilallo) [inline src/components/Placeholder.js]: taken from Slate's default styles

File: src/components/Placeholder.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+  width: 0;
+  white-space: nowrap;
+  float: left; // https://github.com/ianstormtaylor/slate/issues/1436
```
Comment (jorilallo) [inline src/components/Placeholder.js]: this bad boy fixes the cursor jumping

File: src/index.js [hunk 3]
```diff
@@ -210,6 +210,18 @@ class RichMarkdownEditor extends React.PureComponent<Props, State> {
     return renderNode(props);
   };
 
+  renderPlaceholder = (props: SlateNodeProps) => {
```
Comment (jorilallo) [inline src/index.js]: lifted from Slate's code. Not sure if the best palce

File: src/index.js [hunk 4]
```diff
@@ -210,6 +210,18 @@ class RichMarkdownEditor extends React.PureComponent<Props, State> {
...
+    const { editor, node } = props;
+    if (!editor.props.placeholder) return;
+    if (editor.state.isComposing) return;
```
Comment (tommoor) [inline src/index.js]: Do you remember where this `isComposing` is from? Can't find it anywhere.

Comment (tommoor) [reply src/index.js]: Ah, I see the whole method is from Slate's source.

**Comment by tommoor** [comment]
Now on master

**Pull request closed without merging**